inst/NEWS.Rd

```diff
@@ -1,5 +1,15 @@
 \name{NEWS}
 \title{NEWS for GENESIS}
+\section{Version 2.19.9}{
```

Comment:
This should actually be version 2.21.1 after last release

Reply:
Fixed